conf/routes

```diff
@@ -21,7 +21,7 @@ POST    /api2/atoms/:id/assets          controllers.Api2.addAsset(id)
 
 PUT     /api2/atom/:id/publish          controllers.Api2.publishMediaAtom(id)
 PUT     /api/atom/:id/update-metadata   controllers.Api2.updateMetadata(id)
-POST    /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
+PUT     /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
```

Comment:
I've created a PR with the CDS change: https://github.com/guardian/content_delivery_system/pull/50

As for the deploy order, would it be alright to deploy this first and CDS right after?